app/services/dashboard/data.rb

```diff
@@ -98,8 +102,20 @@ def reports_missing
 
     def out_of_sync_enabled?
       return true unless settings[:origin]
-      setting = Setting[:"#{settings[:origin].downcase}_out_of_sync_disabled"]
-      setting.nil? ? true : !setting
+      if for_all_origin?
+        host_origins.map { |origin| out_of_sync_setting_for(origin) }.any?
```

Comment:
you can pass the block directly to `any?` (https://ruby-doc.org/core-2.5.3/Enumerable.html#method-i-any-3F): 
```suggestion
        host_origins.any? { |origin| out_of_sync_setting_for(origin) }
```